index.js

```diff
@@ -41,7 +47,20 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  if (movies.length === 0) {
+    return 0;
+  }
+  let firstMetascore = Number(movies[0].metascore);
+  for (let i = 0; i < movies.length; i++){
+    if (Number(movies[i].metascore) > firstMetascore){
+      firstMetascore = Number(movies[i].metascore);
+    }
+  }
+  return firstMetascore;
+
+
```

Comment:
Great work Sabrina!  My only question is how can we harden line 50 so that falsy values and empty objects also return 0?